tests/network/network_policy/test_network_policy.py

```
@@ -133,14 +132,12 @@ def test_network_policy_deny_all_http(
-            host=network_policy_vmb.ssh_exec,
-            commands=[
-                shlex.split(f"curl --head {dst_ip}:{port} --connect-timeout {CURL_TIMEOUT}")
-                for port in [PORT_80, PORT_81]
-            ],
-        )
+    with pytest.raises(SSHCommandError):
+        for port in [PORT_80, PORT_81]:
+            run_ssh_commands(
+                vm=network_policy_vmb,
+                commands=shlex.split(f"curl --head {dst_ip}:{port} --connect-timeout {CURL_TIMEOUT}"),
+            )
```

Ruff (0.14.13)

`pytest.raises()` block should contain a single simple statement

(PT012)